newIDE/app/src/InstancesEditor/InstancesResizer.js

```diff
@@ -210,14 +211,17 @@ export default class InstancesResizer {
       : roundedTotalDeltaX;
     const flippedTotalDeltaY = isTop ? -roundedTotalDeltaY : roundedTotalDeltaY;
 
+    // While rezising both sides, TotalDelta must be doubled
+    const altOffset = centeredResize ? 2 : 1;
```

Comment:
altOffset is not very meaningful as a variable name. Can you think of something that represents this factor?